ui/src/alerting/actions/checks.ts

```diff
@@ -33,9 +33,9 @@ import {
   GetState,
   RemoteDataState,
   CheckViewProperties,
-  Label,
   PostCheck,
 } from 'src/types'
+import {Label} from 'src/client'
```

Comment:
if you're for serious about this, it'd be easier to just import the Label definition into 'src/types' instead